src/instruction_handlers/precompiles.rs

```diff
@@ -66,16 +66,20 @@ fn precompile_call<T: Tracer, W>(
         let heaps = &mut vm.state.heaps;
         match address_low {
             KECCAK256_ROUND_FUNCTION_PRECOMPILE_ADDRESS => {
-                keccak256_rounds_function::<_, false>(0, query, heaps);
+                vm.state.cycle_counts.keccak256_cycles +=
+                    keccak256_rounds_function::<_, false>(0, query, heaps).0;
```

Comment:
Dumb question / maybe was discussed already: Would adding cycle tracking into the tracer functionality be inferior? I.e., something like this:

```rust
pub trait Tracer {
    // ...
    fn after_precompile(&mut self, precompile_stats: PrecompileStats) {}
}

pub enum PrecompileStats {
    Keccak256(usize),
    Sha256(usize),
    EcRecover(usize),
    Secp256k1Verify(usize),
    Decommit(usize),
}
```